.envrc

```diff
@@ -1,28 +1,25 @@
 # PostgreSQL
-export TEST_POSTGRES_BASE_URI="postgres://prisma:prisma@localhost:5432"
 export TEST_POSTGRES_URI="postgres://prisma:prisma@localhost:5432/tests"
-# Note: the isolated instance is only needed for one test (client/src/__tests__/integration/errors/connection-limit-postgres/test.ts)
-export TEST_POSTGRES_ISOLATED_URI="postgres://prisma:prisma@localhost:5435/tests"
 export TEST_POSTGRES_URI_MIGRATE="postgres://prisma:prisma@localhost:5432/tests-migrate"
```

Comment:
```suggestion
# Note that the database name must be `tests` (in our test setup we search and replace that value later)
export TEST_POSTGRES_URI_MIGRATE="postgres://prisma:prisma@localhost:5432/tests-migrate"
```

Reply:
Decided to add a note at top of file instead as otherwise we need to repeat this one too many times all over the files